src/streams/StreamItem.js

```diff
@@ -151,13 +160,23 @@ export default function StreamItem(props: Props): Node {
       }}
     >
       <View style={wrapperStyle}>
+        {isCollapsible === true && (
+          <Touchable onPress={handleCollapse && (() => handleCollapse(streamId))}>
```

Comment:
This logic is a little bit complicated. One thing that would simplify it would be to pass "this should show the expand/collapse UI" and "here's the callback for the expand/collapse UI" as a single prop, rather than two props.

That helps because all the information carried by `isCollapsible` is already conveyed in `handleCollapse`: `isCollapsible` should be truthy just when `handleCollapse` is. IOW we only need `handleCollapse`, not a separate `isCollapsible`.